Extensions/SendTo/InstructionsViewController.swift

```diff
@@ -57,11 +57,11 @@ func setupHelpView(view: UIView, introText: String, showMeText: String) {
     let label2 = UILabel()
     view.addSubview(label2)
     label2.numberOfLines = 0
-    label2.lineBreakMode = NSLineBreakMode.ByWordWrapping
+    label2.lineBreakMode = NSLineBreakMode.byWordWrapping
     label2.font = InstructionsViewControllerUX.TextFont
     label2.textColor = InstructionsViewControllerUX.TextColor
-    label2.textAlignment = NSTextAlignment.Center
-    label2.attributedText = highlightLink(showMeText, withColor: InstructionsViewControllerUX.LinkColor)
+    label2.textAlignment = NSTextAlignment.center
+    label2.attributedText = highlightLink(showMeText as NSString, withColor: InstructionsViewControllerUX.LinkColor)
```

Comment:
Does `highlightLink` need to take `NSString` or can we convert this to `String`?